gcovr/tests/simple1/reference/coverage.xml

```diff
@@ -1,7 +1,7 @@
-<?xml version="1.0" ?>
+<?xml version="" ?>
```

Comment:
To double check, is this correct? 
All other files with changes have the XML version added, this one has it _removed_...

Reply:
@wolf99 The files are sanitized by a few simple regexes, one of which removes the gcovr version. The files are edited on-disk, so the sanitized files ought to be committed.

Unfortunately this regex is a bit overzealous and also mangles the XML declaration. The result is *technically* not a well-formed XML document.

I've addressed those issues in #188, so that this PR can move forward independently.